rust/src/builtins/apply_live.rs

```diff
@@ -55,18 +55,20 @@ pub(crate) fn applylive_entrypoint(args: &Vec<String>) -> CxxResult<()> {
 
     let args = get_args_variant(sysroot, opts)?;
 
-    let params = variant_utils::new_variant_tuple(&[args]);
+    let params = Variant::from_tuple(&[args]);
     let reply = &client.get_os_ex_proxy().call_sync(
         "LiveFs",
         Some(&params),
         gio::DBusCallFlags::NONE,
         -1,
         gio::NONE_CANCELLABLE,
     )?;
-    let reply_child =
-        variant_utils::variant_get_child_value(reply, 0).ok_or_else(|| anyhow!("Invalid reply"))?;
+    if reply.n_children() != 1 {
```

Comment:
The original code was checking for `< 1`. I think it is ok to keep this new check, but at the same time it could be worth recording `n_children` in the error message.